File: samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/Services/BotSettings.cs
﻿// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using System.Collections.Generic;
using Microsoft.Bot.Builder.Azure;
using Microsoft.Bot.Builder.Skills.Models.Manifest;
using Microsoft.Bot.Builder.Solutions;

namespace VirtualAssistantSample.Services
{
    public class BotSettings : BotSettingsBase
    {
        public List<SkillManifest> Skills { get; set; } = new List<SkillManifest>();
    }
}

File: templates/typescript/generator-botbuilder-assistant/generators/app/templates/sample-assistant/src/services/botSettings.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */
import { IBotSettingsBase, ISkillManifest } from 'botbuilder-solutions';

export interface IBotSettings extends IBotSettingsBase {
    skills: ISkillManifest[];
}
